### PR TITLE
Update package.json to include repo name and other meta info. Fixes #30

### LINKED ISSUE #30: package.json should point to this repo in the repository field.
I got to this repo from the readme, but if it was also in package.json it would tie the two together in the usual way.

### PR DESCRIPTION
**Pull request by karanjthakkar**

### CODE REVIEW

**Comment by karanjthakkar** [comment]
I signed it!

File: package.json [hunk 1]
```diff
@@ -2,6 +2,14 @@
   "name": "web-push-encryption",
   "version": "2.0.0",
   "description": "A library that handles payload encryption for the Web Push protocol",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/GoogleChrome/web-push-encryption.git"
```
Comment (gauntface) [inline package.json:7]: I've not seen this git+HTTPS syntax before what does it do?

**Comment by gauntface** [comment]
@wibblymat what do you think? Apart from the one question this is LGTM from me

File: package.json (hunk 1 above)
Comment (karanjthakkar) [inline package.json:7]: @gauntface This is what `npm init` does by default. You can also see a similar usage: `https://github.com/GoogleChrome/push-notifications/blob/master/package.json`

**Comment by wibblymat** [comment]
Yep, looks good!

**Pull request merged**